src/org/zaproxy/zap/extension/websocket/ui/WebSocketMessagesViewFilter.java

```diff
@@ -159,6 +196,16 @@ private String toString(boolean shouldIncludeValues) {
 			}
 		}
 
+		if (payloadFilter != null){
+			empty = false;
+			sb.append(Constant.messages.getString("websocket.filter.label.pattern"));
+			if (shouldIncludeValues) {
+				sb.append(": " + payloadFilter.getStringPayloadPattern() + " ");
```

Comment:
It was better as before (no need to create another `StringBuilder`):
```Java
sb.append(": ");
sb.append(payloadFilter.getStringPayloadPattern());
sb.append(' ');
```
was it changed because of a specific reason?

Reply:
The result is the same, but as is now it ends up creating another `StringBuilder`/`String` unnecessarily.

Reply:
@ManosMagnus IIRC StringBuilder appends are more efficient than plus sign concatenation. Since a StringBuilder is already in use may as well leverage it fully.

If it helps somehow I believe you can chain appends, for example: `sb.append(": ").append(payloadFilter.getStringPayloadPattern()).append(' ');`

Reply:
This is the last review comment pending action.